metrics/ci_integrations/lib/destination/firestore/client_factory/firestore_destination_client_factory.dart

```diff
@@ -11,13 +12,34 @@ class FirestoreDestinationClientFactory
     implements
         DestinationClientFactory<FirestoreDestinationConfig,
             FirestoreDestinationClientAdapter> {
-  const FirestoreDestinationClientFactory();
+  /// The [FirebaseAuth] instance used to authenticate the Firestore client.
+  /// 
+  /// If this is `null` then the [FirebaseAuth.initialize] is used.
```

Comment:
It looks like this should be rather a constructor description rather than a private field.